charts/aac-manage-case-assignment/values.preview.template.yaml

```diff
@@ -21,7 +21,7 @@ java:
           alias: NOTIFY_MCA_API_KEY
   environment:
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-1260-java.ccd
-    CCD_DEFINITION_STORE_API_BASE_URL: http://ccd-definition-store-api-pr-575-java.ccd
+    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-2267-java.ccd
```

Comment:
Revert after def-store and data-store merged

Reply:
Reverted